deploy/src/main/deploy/helm/templates/dispatch-router/config/qdrouterd-with-broker.yaml

```diff
@@ -66,23 +66,38 @@
   ["connector", {
     "sslProfile": "external",
     "name": "broker",
-    "host": "{{ .Release.Name }}-artemis",
     "port": 5671,
     "role": "route-container",
+{{- if .Values.azure.serviceBus.enabled }}
```

Comment:
It might make more sense to create and use completely new configuration (qrouterd-with-azure) than to template this one. It might get complicated to see what it actually resolves to in different scenarios.

Reply:
Good idea, I will change that. 👍